src/assets/fonts.css

```diff
@@ -1,45 +1,81 @@
-@font-face {
-  font-family: 'Inter';
-  font-style: normal;
-  font-weight: 500;
-  font-display: swap;
-  src: local('Inter-Regular'), local('Inter Regular') local('Inter'),
-    url('~assets/fonts/Inter-Regular.ttf') format('ttf');
-}
-@font-face {
-  font-family: 'Inter';
-  font-style: normal;
-  font-weight: 600;
-  font-display: swap;
-  src: local('Inter-SemiBold'), local('Inter SemiBold') local('Inter'),
-    url('~assets/fonts/Inter-SemiBold.ttf') format('ttf');
-}
-@font-face {
-  font-family: 'Inter';
-  font-style: normal;
-  font-weight: 700;
-  font-display: swap;
-  src: local('Inter-Bold'), local('Inter Bold') local('Inter'),
-    url('~assets/fonts/Inter-Bold.ttf') format('ttf');
-}
+/* Variable Fonts */
+@supports (font-variation-settings: normal) {
+  @font-face {
+    font-family: 'Inter';
+    font-style: normal;
+    font-weight: 100 900;
+    font-display: swap;
+    src: url('~assets/fonts/Inter-roman.var.woff2') format('woff2');
+  }
 
-@font-face {
-  font-family: 'JetBrains Mono';
-  font-style: normal;
-  font-weight: 400;
-  font-display: swap;
-  src: local('JetBrainsMono-Regular'), local('JetBrains Mono Regular'),
-    url('~assets/fonts/JetBrainsMono-Regular.woff2') format('woff2'),
-    url('~assets/fonts/JetBrainsMono-Regular.woff') format('woff'),
-    url('~assets/fonts/JetBrainsMono-Regular.otf') format('opentype');
```

Comment:
Are we not using the monospaced font anymore? It was being used in the advanced search query page iirc.

Reply:
We should bring back the monospaced font, so probably it was a mistake.